src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.cpp

```diff
@@ -12,10 +12,10 @@ template <size_t Dim, typename Frame, typename DataType>
 tnsr::aa<DataType, Dim, Frame> compute_spacetime_metric(
     const Scalar<DataType>& lapse, const tnsr::I<DataType, Dim, Frame>& shift,
     const tnsr::ii<DataType, Dim, Frame>& spatial_metric) noexcept {
-  tnsr::aa<DataType, Dim, Frame> spacetime_metric{
-      make_with_value<DataType>(lapse.get(), 0.)};
+  auto spacetime_metric =
+      make_with_value<tnsr::aa<DataType, Dim, Frame>>(get<>(lapse), 0.);
```

Comment:
@markscheel tested that `get(lapse)` works fine, so no need for the empty angle brackets